util/src/main/java/org/killbill/billing/util/config/definition/KillbillConfig.java

```diff
@@ -16,9 +16,28 @@
  */
 package org.killbill.billing.util.config.definition;
 
+import java.util.List;
+
+import org.killbill.billing.callcontext.InternalTenantContext;
+import org.skife.config.Config;
+import org.skife.config.Default;
+import org.skife.config.Description;
+import org.skife.config.Param;
+import org.skife.config.TimeSpan;
+
 /*
  * Marker interface for killbill config files
  */
 public interface KillbillConfig {
 
+    @Config("org.killbill.rescheduleIntervalOnLock")
+    @Default("30s, 1m, 1m, 3m, 3m, 10m")
+    @Description("Tme delay to reschedule an invoice run when lock is held")
+    List<TimeSpan> getRescheduleIntervalOnLock();
```

Comment:
NACK - `KillbillConfig` is the marker interface for all of our configs. These configs won't apply to `RbacConfig` or `EhCacheConfig` for instance.

You could introduce an intermediate interface though (`LockAwareConfig extends KillbillConfig`).

Reply:
Ok, fair enough, that was my first thought as well -  but then I had convinced myself differently...

Fixed in https://github.com/killbill/killbill/pull/1894/commits/6f37abe76378cba90c91c3695d04411bed84d1b9